README.md

```diff
@@ -1,7 +1,4 @@
-# Spork
+# Sponsored by Tesseract!
```

Comment:
missing an `rr` in `tessera[rr]ct`

Reply:
How dare you, you won't get my collaboration then! CLOSED!